resources/build/history/history-utils.sh

```diff
@@ -1,57 +1,49 @@
 #!/bin/sh
```

Comment:
I think I had to set this to `#!/bin/bash` for it to run on my 18.04 Ubuntu box.
Otherwise, I get a syntax error
> ./history-utils.sh: 30: ./../../../resources/shellHelperFunctions.sh: Syntax error: "(" unexpected